src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionTest.java

```diff
@@ -226,16 +241,23 @@ void listConsumerSubscriptions(VertxTestContext context) throws InterruptedExcep
 
         consumerService()
                 .createConsumer(context, groupId, json)
-                .subscribeConsumer(context, groupId, name, topic)
-                .subscribeConsumer(context, groupId, name, topic2);
-
-        CompletableFuture<Boolean> consume = new CompletableFuture<>();
+                .subscribeConsumer(context, groupId, name, topic, topic2);
 
         // poll to subscribe
+        CountDownLatch latch = new CountDownLatch(2);
         consumerService()
                 .consumeRecordsRequest(groupId, name, BridgeContentType.KAFKA_JSON_JSON)
-                .as(BodyCodec.jsonObject())
-                .send(ar -> consume.complete(true));
+                .as(BodyCodec.jsonArray())
```

Comment:
what's the array we are sending? consuming doesn't have any body

Reply:
Good question. Leaving `jsonObject` causes `Future{cause=Invalid Json Object decoded as io.vertx.core.json.JsonArray}`

Reply:
Ok ok, it's not about body sent, my bad. It's about decoding the response which is actually a JSON array.

Reply:
Yeah, I believe it is assembled in `io.strimzi.kafka.bridge.http.converter.HttpJsonMessageConverter#toMessages`